Source/Serilog.Enrichers.Span/ActivityTagEnricher.cs

```diff
@@ -0,0 +1,31 @@
+namespace Serilog.Enrichers.Span
+{
+    using System.Diagnostics;
+    using System.Linq;
+    using Serilog.Core;
+    using Serilog.Events;
+
+    /// <summary>
+    /// A log event enricher which adds tags from the current <see cref="Activity"/>.
+    /// </summary>
+    public class ActivityTagEnricher : ILogEventEnricher
+    {
+        private const string ActivityTags = "ActivityTags";
```

Comment:
Thoughts on calling this `Tags` or even the Open Telemetry name `Attributes` (See https://rehansaeed.com/deep-dive-into-open-telemetry-for-net/#attributes)?